packages/dropdown/src/react/label.tsx

```diff
@@ -17,15 +17,14 @@ const styles = {
     )
 }
 
-interface LabelProps {
+interface LabelProps extends HTMLPropsFor<'label'> {
   label?: ReactNode
-  inputId: string
 }
 
 export const Label: FC<LabelProps> = props => {
   const themeName = useTheme()
   return (
-    <label htmlFor={props.inputId}>
```

Comment:
Think I should put this back

Reply:
i don't think using the `for` attr works to reference a `button` element. using this [example](https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html) aria-labelledby seems to be the right approach. aria-labelledby should reference the label and the button.